apps/arweave/src/ar_chunk_storage.erl

```diff
@@ -34,6 +34,9 @@
 
 encode_packing({spora_2_6, Addr}) ->
 	"spora_2_6_" ++ binary_to_list(ar_util:encode(Addr));
+encode_packing({composite, Addr, PackingDifficulty}) ->
+	"composite_" ++ binary_to_list(ar_util:encode(Addr)) ++ ":"
```

Comment:
Flagging this discussion here (was started in the other repo). @vird pointed out that `:` is used for address checksum and it will cause a problem on windows (eventually). So perhaps we can select a different delimeter?

Slack discussion here: https://digitalhistor-8tx7838.slack.com/archives/C04Q93X02AJ/p1723576491561449

Maybe we use `.` as the delimiter?

Reply:
I like `.`!